oak-api/src/main/java/org/apache/jackrabbit/oak/api/jmx/IndexStatsMBean.java

```diff
@@ -127,6 +128,15 @@ public interface IndexStatsMBean {
      */
     String getReferenceCheckpoint();
 
+    @Description("Force update the indexing lane to a latest checkpoint. This will abort and pause the running lane, release its lease and set the reference checkpoint to a latest one." +
+            "Any content changes b/w the old reference checkpoint and the new one will be not be indexed and a reindexing would be required." +
```

Comment:
I would prefer to not use abbreviations in user facing documentation, so use `between` instead of `b/w`. I'm not even sure that `b/w` is meant to mean `between`, I'm just guessing.